Doesn't work when Prettier is only installed globally

I have `prettier` and `prettier-eslint-cli` installed globally because I want to use it in any JavaScript project with vim.

When I format a file, I get the text of the file replaced with this:

```
prettier-eslint-cli error: There was a problem trying to format the stdin text Error: Cannot find module '../../../../../../path/to/current/project/node_modules/prettier'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at prettify (/usr/local/lib/node_modules/prettier-eslint-cli/node_modules/prettier-eslint/dist/index.js:79:16)
    at format (/usr/local/lib/node_modules/prettier-eslint-cli/node_modules/prettier-eslint/dist/index.js:66:18)
    at _callee2$ (/usr/local/lib/node_modules/prettier-eslint-cli/dist/format-files.js:77:54)
    at tryCatch (/usr/local/lib/node_modules/prettier-eslint-cli/node_modules/regenerator-runtime/runtime.js:64:40)
    at Generator.invoke [as _invoke] (/usr/local/lib/node_modules/prettier-eslint-cli/node_modules/regenerator-runtime/runtime.js:355:22)
    at Generator.prototype.(anonymous function) [as next] (/usr/local/lib/node_modules/prettier-eslint-cli/node_modules/regenerator-runtime/runtime.js:116:21)
```

But when I install `prettier` locally in the project, it works. I feel like it should work either way.

--- a/.all-contributorsrc
+++ b/.all-contributorsrc
@@ -25,6 +25,7 @@
       "avatar_url": "https://avatars3.githubusercontent.com/u/3266363?v=3",
       "profile": "https://github.com/aharris88",
       "contributions": [
+        "code"
         "review"
       ]
     },

--- a/src/parser.js
+++ b/src/parser.js
@@ -14,10 +14,7 @@ const parser = yargs
       default: getPathInHostNodeModules('eslint'),
       describe: 'The path to the eslint module to use',
     },
-    prettierPath: {
-      default: getPathInHostNodeModules('prettier'),
-      describe: 'The path to the prettier module to use',
-    },
+    prettierPath: {describe: 'The path to the prettier module to use'},
     log: {default: false, describe: 'Show logs'},
     // TODO: if we allow people to to specify a config path,
     // we need to read that somehow. These can come invarious

